modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/RasterMBLayer.java

```diff
@@ -225,10 +225,17 @@ public Number getFadeDuration() {
 
         // Use of builder is easier for code examples; but fills in SLD defaults
         // Currently only applies the opacity.
+        if(getBrightnessMax() != null || getBrightnessMin() != null){
```

Comment:
You're using an OR here, but assuming both are not null later. Should add individual null checks and initialize them to their default values in case (for example) max is set but min is not.